aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -138,12 +138,12 @@ void Dispatcher::deregisterDef_(const OperatorHandle& op, const OperatorName& op
   cleanup(op, op_name);
 }
 
-RegistrationHandleRAII Dispatcher::registerImpl(OperatorName op_name, c10::optional<DispatchKey> dispatch_key, KernelFunction kernel) {
+RegistrationHandleRAII Dispatcher::registerImpl(OperatorName op_name, c10::optional<DispatchKey> dispatch_key, KernelFunction kernel, std::unique_ptr<FunctionSchema> inferred_function_schema) {
   std::lock_guard<std::mutex> lock(mutex_);
 
   auto op = findOrRegisterName_(op_name);
 
-  auto kernel_handle = op.operatorIterator_->op.registerKernel(dispatch_key, std::move(kernel));
+  auto kernel_handle = op.operatorIterator_->op.registerKernel(dispatch_key, std::move(kernel), std::move(inferred_function_schema));
```

Comment:
I'm happy to rename this, but I don't think `kernel_handle` naming is a big deal, since I think of the schema as semantically being part of the kernel.

Reply:
Agree it's not a big deal - I think I navigate by these sorts of names more than most while I'm learning code, otherwise I doubt I'd even have noticed it :)

Reply:
This is now `handle`